### ISSUE TITLE
Modify roles implementation to permit Admin role to create repos

### ISSUE THREAD

**Issue by pwilkins** [mitodl/lore#149]
As currently implemented, the `Admin` role can't have the permission to create repositories, since the repository has to exist before the `Admin` role is created.  This issue identifies the need to change the implementation to allow the `Admin` role to create repositories.

**Comment by Ferdi** [comment]
I didn't think "Admin" would have permission to create repositories.   "Admin", "Curator", and "Author" are roles created for each repository.
There should be (is?) a global role that will be able to create repositories.

@giocalitri is that correct ?

**Comment by giocalitri** [comment]
@Ferdi, yes, that's correct. My idea is that there should be an extra group called "Lore_admin" or "Repo_creator" that has the permission `learningresources.add_repository`

**Comment by pdpinch** [comment]
The chart in #26 shows admin having permission to create repositories. Should we update that?

**Comment by pdpinch** [comment]
@giocalitri how did we end up resolving this?

**Comment by giocalitri** [comment]
we never solved: we created a group for lore admins who have rights to create repositories
